docs/components/ApplePayButton.md

```diff
@@ -4,7 +4,7 @@ title:ApplePayButton
 ---
 Renders an Apple Pay button to use in the Square Payment Form, pre-styled to meet Apple's Human Interface Guidelines.
 ## Props
-|Name|Type|Description|
-|---|---|---|
-|loadingView|React.ReactNode|Placeholder view when the Apple Pay is being initialized|
-|unavailableView|React.ReactNode|Placeholder view when Apple Pay is not available|
\ No newline at end of file
+|Name|Type|Description|Default Value|
+|---|---|---|---|
+|loadingView|ReactReactNode|Placeholder view when the Apple Pay is being initialized||
```

Comment:
Should this one stay the same? `React.ReactNode`

Reply:
Done!